packages/web3-eth-abi/src/index.js

```diff
@@ -24,7 +24,7 @@
 var _ = require('underscore');
 var utils = require('web3-utils');
 
-var EthersAbi = require('ethers/utils/abi-coder').AbiCoder;
+var EthersAbi = require('../node_modules/ethers/utils/abi-coder').AbiCoder;
```

Comment:
Is this necessary?

Leaving a suggested code comment here based on the discussion below...
```suggestion
var EthersAbi = require('../node_modules/ethers/utils/abi-coder').AbiCoder; // Rollup optimization
```

Reply:
Yes, because we would otherwise bundle the whole ethers lib in-to web3.js.

Reply:
Ok maybe add a comment saying this optimizes the build?

Reply:
I will add a smaller section to the root README file about this because it required smaller dependency related improvements for the build in other packages as well and in addition, is it good to have the "concept" behind it written down somewhere for potential contributors.

Reply:
I think your idea for adding a developer's guide in the README is excellent. It would be very helpful to anyone contributing to the project esp. if they are proposing publication changes or writing tests, etc. 

Above I've left a suggestion that a comment be put in the code because someone might come across this much later and try to 'fix' it.

Reply:
I've added a section to the root README file and added the required comment to the ``index.js`` file of the ``web3-eth-abi`` package.

Reply:
The new comment is very clear! Great.

Reply:
I don't get why `ethers/utils/abi-coder` will bundle the whole thing. I will do a test later today.